### DISCUSSION TITLE
Ability to select multiple photos

### DISCUSSION THREAD

**Discussion by TechieDylan** [Radiokot/photoprism-android-client#41] [Ideas]
Right now using google photos for example when I need to upload multiple images to say an email I can press and hold on an image to start check marking which ones I want makes it a bit more of a smooth experience (image below for reference). 

Where right now in the app you have to select the picture then go back into the gallery and select the next image. Not sure if this kind of functionality is possible but would make for a big improvement in my use cases. 

![image](https://github.com/Radiokot/photoprism-android-client/assets/79419027/a1d8b1df-1920-4240-b800-1d86b4c594d5)

**Comment by Radiokot** [comment]
Hi Dylan, it is possible and I'm planning to do it (in the selection mode)

**Comment by Radiokot** [comment]
Dylan, please check out the latest release with this feature:
https://github.com/Radiokot/photoprism-android-client/releases/tag/1.8.0(15)

**Reply by TechieDylan** [reply]
Just installed it, absolutely what I'm looking for. Is it possible to have it selectable when you just open the app as well.

Ex. open PhotoPrism Gallery, then hold on an image to bring up the selection or always just have the selection overlay there so you could tap the circle to select an image or tap the rest of the image to open an image

Is something like that possible as well this way you can select multiple images then press a share button and select an app to put it into?

**Reply by Radiokot** [reply]
It is indeed yet another one of the most convenient ways to send files. I'm planning to do it as well.

**Reply by ippocratis** [reply]
> Ex. open PhotoPrism Gallery, then hold on an image to bring up the selection or always just have the selection overlay there so you could tap the circle to select an image or tap the rest of the image to open an image
> 
This actually will be more useful.
Multiple selection from within photoprism gallery and then select the app you want to share with. 
Some apps (e.g. viber) refuse to bring up anything else than the default file manager in their internal file picker , thus photoprism gallery cant be selected as the file picker.

**Reply by Radiokot** [reply]
Hi.
The multiple selection feature is available in the latest release:
https://github.com/Radiokot/photoprism-android-client/releases/tag/1.15.0(23)
